stdlib/array.gr

```diff
@@ -768,7 +789,7 @@ provide let unique = array => {
  * @param array2: The array to provide values for the second tuple element
  * @returns The new array containing indexed pairs of `(a, b)`
  *
- * @throws Failure(String): When the arrays have different sizes
+ * @throws IndexOutOfBounds(String): When the arrays have different sizes
```

Comment:
```suggestion
 * @throws IndexOutOfBounds: When the arrays have different sizes
```
IndexOutOfBounds doesn't carry data, right?

Reply:
That is true, this for catching that.